.github/workflows/ci.yml

```diff
@@ -162,9 +162,17 @@ jobs:
     runs-on: ubuntu-latest
     services:
       postgres:
-        image: postgres:11.5-alpine
+        image: postgres:15.6-alpine
         ports:
         - 5433:5432
+        env:
```

Comment:
[This was already present](https://github.com/hypothesis/lms/pull/6201/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR72) for functest but not these